internal/storage/service.go

```diff
@@ -199,6 +208,7 @@ func (s *serviceImpl) SubmitAip(
 	if err != nil {
 		return nil, err
 	}
+	bucket = blob.PrefixedBucket(bucket, "aips/")
```

Comment:
The same, I think having `aips/` as a constant may be beneficial. It's used in three different places here, safer to update a constant than three strings if we decide to change the prefix at some point.